src/lib.rs

```diff
@@ -568,6 +568,46 @@ impl<'a> Record<'a> {
 }
 
 /// Metadata about a log message.
+///
+/// # Use
+///
+/// `Metadata` structs are created when users of the library use
+/// logging macros.
+///
+/// They are consumed by implementations of the `Log` trait in the
+/// `enabled` method. 
+///
+/// `Record`s use `Metadata` to determine the log message's severity
+/// and target.
+///
+///
+/// Users should use the `log_enabled!` macro in their code to avoid
+/// constructing expensive log messages.
+///
+/// ## Examples
```

Comment:
Please use a full-sized heading like we do [elsewhere](https://github.com/rust-lang-nursery/log/blob/50ce9a648f2f0c8089cf52e32d78c2741d3ac3ce/src/macros.rs#L54).